core/src/main/java/org/mskcc/cbio/oncokb/util/EvidenceUtils.java

```diff
@@ -986,8 +986,10 @@ public static void annotateEvidence(Evidence evidence) throws ParserConfiguratio
                     Article tempAT = articleBo.findArticleByPmid(tempPMID);
                     if(tempAT == null){
                         Article newArticle = NcbiEUtils.readPubmedArticle(tempPMID);
-                        articleBo.save(newArticle);
-                        annotatedArticles.add(newArticle);
+                        if(newArticle != null) {
```

Comment:
Something we should even think about the in the future is that if something failed to be imported (for instance in here, the PMID does not exist), we need to notify the curation teams.